plenum/server/primary_selector.py

```diff
@@ -249,28 +261,35 @@ def is_behind_for_view(self) -> bool:
         # Checks if the node is currently behind the accepted state for this
         # view, only makes sense to call when the node has an acceptable
         # view change quorum
-        _, accepted_ledger_summary = self.has_sufficient_same_view_change_done_messages
-        for (_, own_ledger_size, _), (_, accepted_ledger_size, _) in \
+        _, accepted_ledger_summary = self.get_sufficient_same_view_change_done_messages()
+        for (ledgerId, own_ledger_size, _), (_, accepted_ledger_size, _) in \
                 zip(self.ledger_summary, accepted_ledger_summary):
             if own_ledger_size < accepted_ledger_size:
-                print(own_ledger_size, accepted_ledger_size)
+                logger.debug("{} ledger {} sizes are differ: own {} accepted {}"
+                             "".format(self, ledgerId, own_ledger_size, accepted_ledger_size))
                 return True
         return False
 
     def _start_selection(self):
 
-        error = self._verify_view_change()
```

Comment:
Why did we remove this method?

Reply:
explicit conditionals here seems as more clean and readable solution, at least until future refactoring